shared/js/ui/templates/top-blocked-truncated-list-items.es6.js

```diff
@@ -0,0 +1,31 @@
+const bel = require('bel')
+// TODO/FIXME: this is a hack:
+const _constants = require('./../../../data/constants.js')
+const majorTrackingNetworks = window.constants.majorTrackingNetworks
+
+module.exports = function (companyListMap) {
+   function getScssTrackerName(trackerName) {
```

Comment:
This can be exported to shared/ as a standalone module when/if we want to render the generic icon in the trackers subview too - or anywhere else

Reply:
One thing I'd like to standardize -- can we put the helper functions like this beneath the main `return` block for the module? That way, as other developers glance at these templates they are seeing the main logic for the template vs. the helpers.